R/mash.R

```diff
@@ -36,6 +38,8 @@ mash = function(data,
                 add.mem.profile = FALSE,
                 algorithm.version = c("Rcpp","R"),
                 pi_thresh = 1e-10,
+                posterior_samples = 0,
```

Comment:
@pcarbo am not sure if I like this interface -- having `posterior_samples` and `seed` here; most awkwardly, `seed`. In general i'm still trying to get used to the "workhorse" function based software design which might seem natural to you, thus not a concern here.

Reply:
@gaow Thanks for checking in about this. Don't worry about it for now. It was a suggestion only.